multicloud-ts-buckets/README.md

```diff
@@ -0,0 +1,67 @@
+# Multi-Cloud in AWS and GCP using TypeScript
```

Comment:
We recently added a button to each example to automatically “new” it. We should add that here as well (it will likely also require adding some default config in Pulumi.yaml).

Reply:
Is it ok to have configuration items without a default value? It doesn’t seem like there is an acceptable default for GCP project.

Reply:
Looks like the other GCP examples list project in the template with no default, which is what I have done here too.